### PR TITLE
Doc-514 Install RedisInsight Desktop Client revision

### PR DESCRIPTION
**Pull request by kaitlynmichaelredislabs**

### CODE REVIEW

File: content/ri/installing/install-redis-desktop.md [hunk 1]
```diff
@@ -1,48 +1,53 @@
...
+```
 
-        ```sh
-        chmod +x redisinsight-<platform>-<version>
-        ```
+1. Start RedisInsight.
 
-    - For Windows and MacOS - Run the installer.
+```sh
+/redisinsight-<platform>-<version>
```
Comment (banker) [inline content/ri/installing/install-redis-desktop.md:32]:
```suggestion
./redisinsight-<platform>-<version>
```

**Review by banker — APPROVED** [review]
Looks good. I left one comment.

**Comment by kaitlynmichaelredislabs** [comment]
Still want to add more detail after testing on my Ubuntu VM

**Comment by kaitlynmichaelredislabs** [comment]
@K-Jo @nishantgeorge please take a look at the system requirements table I added here with the information from https://github.com/RedisLabs/redislabs-docs/pull/1213.

**Review by nishantgeorge — APPROVED** [review]
LGTM

Here's the link to the staging environment: https://docs.redislabs.com/staging/DOC-514/ri/installing/install-redis-desktop/

**Pull request merged**